server/api.js

```diff
@@ -139,26 +139,63 @@ router.get("/buyer-tender", async (req, res) => {
 	const buyerId = 1;
 	let page = parseInt(req.query.page) || 1;
 	const itemsPerPage = 25;
-
 	const offset = (page - 1) * itemsPerPage;
+	const totalBuyerTenders = await db.query("SELECT COUNT(buyer_id) FROM bid WHERE buyer_id = $1", [buyerId]);
+	const totalPages = Math.ceil(totalBuyerTenders.rows[0].count / itemsPerPage);
 	const result = await db.query("SELECT * FROM tender WHERE buyer_id = $1 LIMIT $2 OFFSET $3", [buyerId, itemsPerPage, offset]);
+
 	result
-		? res.send(result.rows)
+		? res.send({
+			"result": result.rows, "pagination": {
+				"itemsPerPage": 25,
+				"currentPage": 1,
+				"totalPages": totalPages,
+			},
+		})
 		:
 		res.status(500)
 			.send({ code: "SERVER_ERROR" });
 });
 
-
 router.get("/bidder-bid", async (req, res) => {
-	const bidderId = 1;
 	let page = parseInt(req.query.page) || 1;
+	const bidderId = 1;
 	const itemsPerPage = 25;
-
+	const totalBiddings = await db.query("SELECT COUNT(bidder_id) FROM bid WHERE bidder_id = $1", [bidderId]);
+	const totalPages = Math.ceil(totalBiddings.rows[0].count / itemsPerPage);
 	const offset = (page - 1) * itemsPerPage;
+
 	const result = await db.query("SELECT * FROM bid WHERE bidder_id = $1 LIMIT $2 OFFSET $3", [bidderId, itemsPerPage, offset]);
 	result
-		? res.send(result.rows)
+		? res.send({
+			"result": result.rows, "pagination": {
+				"itemsPerPage": 25,
+				"currentPage": 1,
+				"totalPages": totalPages,
+			},
+		})
+		:
+		res.status(500)
+			.send({ code: "SERVER_ERROR" });
+});
+
+router.get("/single-tender", async (req, res) => {
```

Comment:
I'm not sure that I understand this endpoint. If this is an endpoint to retrieve a tender, then it should be:

```
/tender/:id
```

And pagination doesn't apply. Use the same format as for endpoint that return a single result:

```
{
  resource: {
    ...
  }
}
```

Furthermore, this new endpoint doesn't belong in this PR. Create a new PR for it.